types/lazy/index.d.ts

```diff
@@ -0,0 +1,2 @@
+export * from '../lib'
+export { default } from '../lib'
```

Comment:
Newline at the end of this file would match the rest of the codebase 👍

Reply:
Yeah, I saw that. Pushing again.